src/adhocracy_core/versions.cfg

```diff
@@ -50,8 +49,9 @@ repoze.sendmail = 4.2
 requests = 2.5.1
 rubygemsrecipe = 0.2.1
 selenium = 2.44.0
-sphinxcontrib-napoleon = 0.2.9
-sphinxcontrib-zopeext = 0.2.1
+Babel = 1.3
+repoze.sphinx.autointerface = 0.7.1
+snowballstemmer = 1.2.0
```

Comment:
> Interesting, where do we need that one? (Just curious.)
Babel/snowballstemmer  is just a Sphinx dependency.
spincontrib-napolen is included in Spinx > =3
sphinxcontrib-zopeext is replaced by repoze.sphinx.autointerface